mypy.ini

```diff
@@ -10,82 +10,173 @@ warn_unreachable = True
 local_partial_types = True
 no_implicit_optional = True
 
-# To find all folders that pass mypy you run:
-#
-#   find synapse/* -type d -not -name __pycache__ -exec bash -c "mypy '{}' > /dev/null"  \; -print
-
 files =
   scripts-dev/sign_json,
```

Comment:
can we also have:
 * the rest of `scripts-dev`
 * `scripts`
 * `.ci/scripts`
 * `synctl`

?

Reply:
Let's do that in a follow-up.

Things get weird when running mypy over those, since they're standalone scripts and not part of a larger, single Python package.